[2023-11-14T10:47:36.153+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-14T10:47:32.805852+00:00 [queued]>
[2023-11-14T10:47:36.164+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rand_user.is_api_available manual__2023-11-14T10:47:32.805852+00:00 [queued]>
[2023-11-14T10:47:36.164+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T10:47:36.165+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2023-11-14T10:47:36.165+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-14T10:47:36.183+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-11-14 10:47:32.805852+00:00
[2023-11-14T10:47:36.190+0000] {standard_task_runner.py:55} INFO - Started process 4350 to run task
[2023-11-14T10:47:36.196+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rand_user', 'is_api_available', 'manual__2023-11-14T10:47:32.805852+00:00', '--job-id', '471', '--raw', '--subdir', 'DAGS_FOLDER/predictions.py', '--cfg-path', '/tmp/tmpzyekt_8_']
[2023-11-14T10:47:36.198+0000] {standard_task_runner.py:83} INFO - Job 471: Subtask is_api_available
[2023-11-14T10:47:36.265+0000] {task_command.py:376} INFO - Running <TaskInstance: rand_user.is_api_available manual__2023-11-14T10:47:32.805852+00:00 [running]> on host 91f7624ceef9
[2023-11-14T10:47:36.335+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Gaurav
AIRFLOW_CTX_DAG_ID=rand_user
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-11-14T10:47:32.805852+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-14T10:47:32.805852+00:00
[2023-11-14T10:47:36.336+0000] {http.py:120} INFO - Poking: api/
[2023-11-14T10:47:36.346+0000] {base.py:71} INFO - Using connection ID 'rand_user' for task execution.
[2023-11-14T10:47:36.348+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-11-14T10:47:37.482+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-11-14T10:47:37.538+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=rand_user, task_id=is_api_available, execution_date=20231114T104732, start_date=20231114T104736, end_date=20231114T104737
[2023-11-14T10:47:37.657+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-14T10:47:37.928+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
